crates/wasm_interp/src/lib.rs

```diff
@@ -2,8 +2,48 @@ mod call_stack;
 mod instance;
 pub mod test_utils;
 mod value_stack;
+pub mod wasi;
+
+// Main external interface
+pub use instance::Instance;
 
 // Exposed for testing only. Should eventually become private.
 pub use call_stack::CallStack;
-pub use instance::{Action, Instance};
+pub use instance::Action;
 pub use value_stack::ValueStack;
```

Comment:
I don't really want these types to be in the public API of the crate.
I might do another PR to move all the tests under `src` instead. The tests are looking at internals, not just the public API, so that seems to make more sense.

Reply:
https://github.com/roc-lang/roc/pull/4656